src/texture.cr

```diff
@@ -3,6 +3,10 @@ module SDL
     def initialize(@texture : LibSDL::Texture*)
     end
 
+    def initialize(renderer : Renderer | LibSDL::Renderer, width, height, pixel_format = LibSDL::PixelFormatEnum::RGBA8888, texture_access = LibSDL::TextureAccess::STREAMING)
+      @texture = LibSDL.create_texture(renderer, LibSDL::PixelFormatEnum::RGBA8888, LibSDL::TextureAccess::STREAMING, width, height)
```

Comment:
Typo: you're _not_ passing `pixel_format` and `texture_access`.

Reply:
Good catch, thanks